src/chipper/ui.cljs

```diff
@@ -1,10 +1,15 @@
 (ns chipper.ui
   (:require [chipper.chips :as c]
             [chipper.state :as s]
+            [chipper.notes :as n]
             [goog.string :as gs]
             [goog.string.format]
             [reagent.core :as r]))
 
+
+;; TODO FIXME: everything is relative to :C right now.
```

Comment:
whenever we implement support for transposing we can handle this -- presumably each frame would have an associated key and we could name relative to that or something?